charts/telemetry-collector/templates/configmap.yaml

```
@@ -10,7 +10,7 @@ name: {{ include "accelleran.common.fullname" (dict "top" $) }}-config
 
 data:
   config.yaml: |-
-    {{- if and $.Values.config.internal.influxDB (tpl $.Values.config.internal.influxDB.url $) -}}
+    {{- if not (and $.Values.config.internal.influxDB $.Values.config.internal.influxDB.url) -}}
```

We still would want the `tpl` call in case the injected template would result in an empty string:

```suggestion
    {{- if not (and $.Values.config.internal.influxDB (tpl $.Values.config.internal.influxDB.url $)) -}}
```

---

Ok